jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/container/ContainerConfigurationHandler.java

```diff
@@ -0,0 +1,42 @@
+package io.opentelemetry.contrib.jfr.metrics.container;
+
+import io.opentelemetry.api.metrics.Meter;
+import io.opentelemetry.contrib.jfr.Constants;
+import io.opentelemetry.contrib.jfr.metrics.RecordedEventHandler;
+import jdk.jfr.consumer.RecordedEvent;
+
+public class ContainerConfigurationHandler implements RecordedEventHandler {
+    public static final String EVENT_NAME = "jdk.ContainerConfiguration";
+    public static final String JFR_CONTAINER_CONFIGURATION = "jfr.ContainerConfiguration";
```

Comment:
This doesn't really abide by [our metric name conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-naming).

The `upDownCounterBuilder` name defines the metric name.   Likely this should be something like `cpu.usage` or `jvm.cpu.usage` or `jfr.cpu.usage`.    While metric names aren't stable, if we stick with existing conventions, then based on [existing system conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#metric-instruments) and guidance around runtime, I'd suggest:


Name: `runtime.jvm.cpu.utilization` 
Unit: `1` (guidance for all percentages are to use this unit).